backend-service/Embedded/RelayContainer.py

```diff
@@ -10,19 +13,27 @@ def __init__(self):
         self.relay_container = []
         self.db = Database()
     
-    #lopp through our array and call Relay.toString() 
+    #implement helper method that returns size
+    def get_size(self) -> int:
+        return len(self.relay_container)
+
+    #loop through our array and call Relay.toString() 
     #Implement this method
     def __str__(self):
         for x in self.relay_container:
-            x.toString()
+            x.to_string()
             print("\n")
     
     #user is asking to create a new relay object with given id and boolean state
+
     def addRelay(self, input_id, input_state, name, disctiption):
```

Comment:
I changed the naming conventions for functions, however I think we should be able to change it back if necessary.